bamboo/bootstrap-lint-audit.sh

```diff
@@ -16,13 +19,8 @@ git fetch --all
 if [[ $USE_CACHED_BOOTSTRAP == true ]]; then
   git checkout "$GIT_SHA"
   rm -f package-lock.json
-else
-  git fetch --all
 fi
 
-# Extract cache of compiled TS files
-./bamboo/extract-ts-build-cache.sh
```

Comment:
I'm trying to remember why this was removed but it's coming up as static. Why are we making this change again?

Reply:
Is it just moved down the chain to e.g. lint.sh?

Reply:
More or less.  There's no reason to use the typescript compiled objects as we're not actually *building* anything here.    

Meaning -- audit doesn't need the compiled typescript/bootstrap/etc, it just needs the locks done.        The lint script currently does 

```
# Extract cache of compiled TS files
./bamboo/extract-ts-build-cache.sh

(npm run ci:bootstrap-no-scripts || true) && npm run ci:bootstrap-no-scripts
npm run lint
```

which makes sense at that point -- if the audit is going to fail, no point in building/etc as that just serves to increase the time to seeing the failure.

Reply:
Ah ok, that makes sense. Nice change.